tests/testthat/test-tz-lookup-edge.R

```diff
@@ -0,0 +1,581 @@
+  "Asia/Krasnoyarsk", "Australia/Perth", "America/Bahia", "America/Iqaluit",
+  "Antarctica/McMurdo", "America/Argentina/Ushuaia", "Africa/Tripoli",
+  "Africa/Tripoli", "Asia/Almaty", "Africa/Bamako", "Europe/Saratov",
+  "Antarctica/McMurdo", "America/Vancouver", "Australia/Perth",
+  "Antarctica/Mawson", "Asia/Jayapura", "Australia/Perth", "Asia/Riyadh",
+  "Asia/Yakutsk", "Antarctica/McMurdo", "Asia/Magadan", "America/Nassau",
+  "America/Toronto", "Asia/Yakutsk", "America/Nome", "America/Punta_Arenas",
+  "Africa/Addis_Ababa", "Asia/Yakutsk", "Asia/Kolkata", "Antarctica/McMurdo",
+  "Australia/Perth", "Antarctica/McMurdo", "Antarctica/Vostok",
+  "Asia/Manila", "America/Denver", "Australia/Perth", "Africa/Johannesburg",
+  "Antarctica/Troll", "Africa/Lagos", "Europe/London", "Asia/Shanghai",
+  "Europe/Moscow", "Asia/Ulaanbaatar", "Australia/Perth", "Asia/Shanghai",
+  "America/Godthab", "America/Argentina/Cordoba", "America/Lima",
+  "Asia/Riyadh", "Antarctica/McMurdo", "Antarctica/DumontDUrville",
+  "Antarctica/McMurdo", "Africa/Nouakchott", "Africa/Algiers",
+  "America/Santarem", "Antarctica/Mawson", "Asia/Krasnoyarsk",
+  "Asia/Shanghai", "Australia/Perth", "Africa/Nouakchott", "Europe/Moscow",
+  "Antarctica/McMurdo", "Asia/Irkutsk", "America/Argentina/Ushuaia",
+  "America/St_Johns", "America/Bogota", "Europe/Helsinki", "Antarctica/Vostok",
+  "Australia/Adelaide", "Africa/Khartoum", "Africa/Ndjamena", "Asia/Shanghai",
+  "Asia/Krasnoyarsk", "America/Thule", "Europe/Stockholm", "America/Chicago",
+  "America/Yellowknife", "Europe/Madrid", "Antarctica/McMurdo",
+  "America/Fortaleza", "Antarctica/McMurdo", "America/Santarem",
+  "Europe/Moscow", "Africa/Gaborone", "America/Manaus", "America/Godthab",
+  "Europe/Helsinki", "Africa/Johannesburg", "Antarctica/McMurdo",
+  "Antarctica/Syowa", "Antarctica/McMurdo", "Africa/Dar_es_Salaam",
+  "America/Denver", "Antarctica/Rothera", "Asia/Yekaterinburg",
+  "America/Argentina/Rio_Gallegos", "Antarctica/Syowa", "Europe/Moscow",
+  "Antarctica/Mawson", "Africa/Niamey", "Antarctica/McMurdo", "Antarctica/McMurdo",
+  "Europe/Paris", "Antarctica/McMurdo", "America/Rankin_Inlet",
+  "Asia/Krasnoyarsk", "America/Yellowknife", "Asia/Vientiane",
+  "Antarctica/McMurdo", "America/Toronto", "Antarctica/McMurdo",
+  "Africa/Ouagadougou", "Africa/Harare", "America/Toronto", "America/Fortaleza",
+  "America/Sao_Paulo", "America/Godthab", "Asia/Aqtobe", "Europe/Berlin",
+  "Antarctica/McMurdo", "America/Los_Angeles", "Australia/Darwin",
+  "Africa/Addis_Ababa", "Australia/Perth", "Australia/Perth", "Antarctica/McMurdo",
+  "America/New_York", "America/Belem", "Antarctica/Syowa", "Antarctica/Troll",
+  "Antarctica/McMurdo", "Antarctica/Vostok", "Asia/Yakutsk", "Asia/Vladivostok",
+  "Europe/Bucharest", "America/Iqaluit", "America/Cambridge_Bay",
+  "America/Godthab", "America/Argentina/Salta") -> val
+
+test_that("tz_lookup_coords edge cases work", {
+
+  for (idx in seq_along(lat)) {
```

Comment:
Since this is vectorized, I'm not sure we need to loop through this? I think it could just be:

```r
expect_equal(
      tz_lookup_coords(lat, lng, "fast", FALSE),
      val
    )
```

Reply:
And holy smokes is it ever fast!

Reply:
I'll leave this as is so when/if one fails we can easily pinpoint it